### PR TITLE
Create 2024-03-18-4th-issue-newsletter.md

### PR DESCRIPTION
**Pull request by LK112019**
@matentzn - I think this file is ready for publishing. Thank you!

### CODE REVIEW

**Comment by matentzn** [comment]
Made some formatting updates, but looks good!

File: _posts/_posts/2024-03-18-4th-issue-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+* Updating the [SOP document](https://obofoundry.org/docs/SOP.html) to include a new subsection under "Reviewing Ontologies for OBO Membership." This subsection, titled "Rules of Communication," designates the official ontology reviewer as the authority to decide which community suggestions need to be addressed and which do not, aiming to manage situations where community input might not meet the established principles. This addition is intended to streamline the review process and clarify the role of community feedback in ontology evaluations. For more details, please visit the pull request page #[2528](https://github.com/OBOFoundry/OBOFoundry.github.io/pull/2528)
+* EWG improved the OBO Foundry ontology review process by specifying how community input should be taken into account during the review process. The Ontology reviewer has the last word, and is responsible to communicate which feedback is optional, and which mandatory to be addressed before admission (Issue #[2459](https://github.com/OBOFoundry/OBOFoundry.github.io/issues/2459)).
```
Comment (nlharris) [inline _posts/_posts/2024-03-18-4th-issue-newsletter.md:42]: In the next item you say "The EWG"; I would make this one the same (add "The").

File: _posts/_posts/2024-03-18-4th-issue-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+* A testable solution for supporting prefix declarations in OBO format has been provided by Jim Balhoff. Please help out with testing this important change!: [https://github.com/owlcs/owlapi/pull/1102#issuecomment-1924073439](https://github.com/owlcs/owlapi/pull/1102#issuecomment-1924073439)
+
+---
+
+## Ontologies
+
+### Reviewing Ontologies for OBO Membership
+
+[The Standard Operating Procedure (SOP)](https://obofoundry.org/docs/SOP.html#ROOM:~:text=Ontology%20Acceptance/Rejection%20Decision) for Ontology Acceptance/Rejection Decision. The reviewer presents their assessment and recommendation regarding a new ontology to the OBO Operations Committee. The committee may request clarifications from the reviewer before making a decision on the ontology's acceptance. The decision is made by consensus among the attendees on the call, meaning no strong objections are raised. There is no minimum number of attendees required for the decision to be made.
```
Comment (nlharris) [inline _posts/_posts/2024-03-18-4th-issue-newsletter.md:61]:
```suggestion
We recently updated [the Standard Operating Procedure (SOP)](https://obofoundry.org/docs/SOP.html#ROOM:~:text=Ontology%20Acceptance/Rejection%20Decision) for Ontology Acceptance/Rejection Decision. The reviewer presents their assessment and recommendation regarding a new ontology to the OBO Operations Committee. The committee may request clarifications from the reviewer before making a decision on the ontology's acceptance. The decision is made by consensus among the attendees on the call, meaning no strong objections are raised. There is no minimum number of attendees required for the decision to be made.
```

Comment (nlharris) [reply _posts/_posts/2024-03-18-4th-issue-newsletter.md:61]: This was a sentence fragment

File: _posts/_posts/2024-03-18-4th-issue-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+
+### New ontologies accepted in the OBO Foundry Ontology Library
+ 
+* [The Gall Ontology](https://github.com/adeans/gallont). The GALLONT submission to OBO Foundry, discussed in GitHub issue [#2522](https://github.com/OBOFoundry/OBOFoundry.github.io/issues/2522), proposes an ontology for plant gall phenotypes. It aims to provide a controlled vocabulary for describing plant galls, integrating terms from [PATO](https://obofoundry.org/ontology/pato.html), [PO](https://obofoundry.org/ontology/po.html), and other OBO ontologies. The ontology has been formally accepted and is integrated into the website at the moment.
```
Comment (nlharris) [inline _posts/_posts/2024-03-18-4th-issue-newsletter.md:69]:
```suggestion
* [The Gall Ontology](https://github.com/adeans/gallont). The GALLONT submission to OBO Foundry, discussed in GitHub issue [#2522](https://github.com/OBOFoundry/OBOFoundry.github.io/issues/2522), proposes an ontology for plant gall phenotypes. It aims to provide a controlled vocabulary for describing plant galls, integrating terms from [PATO](https://obofoundry.org/ontology/pato.html), [PO](https://obofoundry.org/ontology/po.html), and other OBO ontologies. The ontology has been formally accepted and is now included on obofoundry.org.
```

File: _posts/_posts/2024-03-18-4th-issue-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+
+
+### New ontologies currently under review 
+
+* [The Space LIfe Sciences Ontology](https://github.com/OBOFoundry/OBOFoundry.github.io/issues/2518) The Space Life Sciences Ontology (LSDAO) submission for OBO Foundry inclusion supports NASA's Life Sciences Data Archive and related systems, offering definitions and organization for space life science research data. It integrates concepts from several OBO Foundry ontologies and the Science Data Discovery Ontology for broader research applicability. Ongoing discussions focus on refining LSDAO for clarity, accuracy, and OBO Foundry compatibility. [ Join the discussion.](https://github.com/OBOFoundry/OBOFoundry.github.io/issues/2518)
+
+### Spotlight on well-established OBO ontologies
+
+In this issue, we continue our ontology spotlight series, highlighting two other ontologies from the OBO Foundry family. [Occupation Ontology (OccO)](https://obofoundry.org/ontology/occo.html) and Food [Ontology (FoodOn)](https://obofoundry.org/ontology/foodon.html)
```
Comment (nlharris) [inline _posts/_posts/2024-03-18-4th-issue-newsletter.md:78]:
```suggestion
In this issue, we continue our ontology spotlight series, highlighting two other ontologies from the OBO Foundry family. [Occupation Ontology (OccO)](https://obofoundry.org/ontology/occo.html) and [Food Ontology (FoodOn)](https://obofoundry.org/ontology/foodon.html)
```

**Review by nlharris — APPROVED** [review]

File: _posts/_posts/2024-03-18-4th-issue-newsletter.md (hunk 1 above)
Comment (matentzn) [reply _posts/_posts/2024-03-18-4th-issue-newsletter.md:42]:
```suggestion
* The EWG improved the OBO Foundry ontology review process by specifying how community input should be taken into account during the review process. The Ontology reviewer has the last word, and is responsible to communicate which feedback is optional, and which mandatory to be addressed before admission (Issue #[2459](https://github.com/OBOFoundry/OBOFoundry.github.io/issues/2459)).
```

**Pull request merged**